modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2788,7 +2788,7 @@ export class Wallet implements IWallet {
           intentType: 'signMessage',
           isTss: true,
           messageRaw: params.message.messageRaw,
-          messageEncoded: params.message?.messageEncoded ?? '',
+          messageEncoded: Buffer.from(params.message?.messageEncoded ?? '').toString('hex'),
```

Comment:
if params.message?.messageEncoded is null, you should throw an error

Reply:
well the point was that future evm chains may not have encoding for message, hence why I made `messageEncoded` optional. If we are to throw an error, might as well make it mandatory.

Reply:
gotcha